backend/scripts/cases_load.py

```diff
@@ -0,0 +1,65 @@
+import csv  # https://docs.python.org/3/library/csv.html
+
+# https://django-extensions.readthedocs.io/en/latest/runscript.html
+
+# python3 manage.py runscript many_load
+import pandas as pd
+import pycountry
+from cases.models import Cases
+
+def run():
+
+    url_cases = 'https://raw.githubusercontent.com/alan-turing-institute/CounterfactualCovid19-inputs/develop/Data/Formatted/Cases_deaths_data_europe.csv'
+    df_cases = pd.read_csv(url_cases)
+
+    countries = df_cases.Country.unique()
+    date = '2020-06-23'
+
+    Cases.objects.all().delete()
+
+    for country in countries:
+
+        ts_data_country_daily, ts_data_country_cum = get_natural_history_data(country, df_cases)
+
+        total_cases = ts_data_country_daily[ts_data_country_daily.index < date].sum()
+
+        try:
+            code_county = pycountry.countries.get(name=country)
+
+            m = Cases(country=country, iso_code=code_county.alpha_3, date=date, cumulative_cases=total_cases)
```

Comment:
I'm not completely sure what we're trying to load in here. Do we want one entry for each date before `2020-06-23`? I think we're only making one entry here, which contains the total number of cases by 2020-06-23.

Reply:
Copying the comment from slack as reference:
  
 For now i just want to try to reproduce the front end map from our data for the first wave, so we have 1 row per country with the total number of cases for the first wave in Europe.